src/main/java/com/fnp/checkexception/custom-checked-exception/customexceptions/ConditionWhichThrowsException.java

```diff
@@ -0,0 +1,11 @@
+package customexceptions;
+
+public class ConditionWhichThrowsException{
+	 
+	  int variable1 = 5;
+	 	public void conditionChecker(int variable2)throws ObservedException{
+			if(variable1 != variable2){
+				throw new ObservedException("The condition does not follow. ");
```

Comment:
I can see you are throwing this unchecked exception..where did you throw the checked CheckException's object.

Reply:
In checked exception we have to throw the exception but in the case of unchecked exception it is not necessary to declare in throws